tests/test_vault.py

```diff
@@ -21,13 +21,21 @@
 
 
 class MockResponse:
-    def __init__(self, json_response={}, status_code=200):
-        self.json_response = json_response
-        self.content = json.dumps(json_response)
+    def __init__(self, json_response=None, status_code=200, content_type='application/json', url='not_given', text=""):
         self.status_code = status_code
+        if json_response is not None:
+            text = json.dumps(json_response)
+        self.content = text
+        self.text = text
+        self.headers = {'content-type': content_type}
+        self.url = url
 
     def json(self):
```

Comment:
where we using this method ?

Reply:
We are using it here: https://github.com/cfconrad/pcw/pull/68/files#diff-7d35fc4e9f2b0c45d5681b1c0bbc34a8R77

Reply:
ah ok I just didn't realize that we overloading this method so I need to check prod code and not test code